servers/rendering/renderer_rd/effects/debug_effects.h

```diff
@@ -72,12 +75,20 @@ class DebugEffects {
 
 	void _create_frustum_arrays();
 
+	/* Combined shadow map */
+	struct {
+		DebugCombinedShadowMapShaderRD shader;
+		RID shader_version;
+		PipelineCacheRD pipeline;
+	} combined_shadow_map;
+
 protected:
 public:
 	DebugEffects();
 	~DebugEffects();
 
 	void draw_shadow_frustum(RID p_light, const Projection &p_cam_projection, const Transform3D &p_cam_transform, RID p_dest_fb, const Rect2 p_rect);
+	void draw_combined_shadow_map(RID p_static_texture, RID p_dynamic_texture, RID p_dest_fb, const Rect2 p_rect);
```

Comment:
```suggestion
	void draw_combined_shadow_map(RID p_static_texture, RID p_dynamic_texture, RID p_dest_fb, const Rect2 &p_rect);
```
I'd pass by reference here